src/serac/physics/utilities/functional/functional.hpp

```diff
@@ -131,13 +131,16 @@ class Functional<test(trial)> : public mfem::Operator {
    * @tparam geometry_dim The dimension of the element (2 for quad, 3 for hex, etc)
    * @tparam spatial_dim The full dimension of the mesh
    * @tparam lambda the type of the integrand functor: must implement operator() with an appropriate function signature
+   * @tparam PointData The type of the data to store for each quadrature point
```

Comment:
`QuadraturePointData` would be more appropriate.

Reply:
`QuadraturePointData` sounds like a good name for the parameter itself, but I think this line of code is documenting the name of the type itself. `data_type`, `data_t`, or just `T` might be good alternatives. We seem to be mixing snake_case (e.g. `geometry_dim`) and CamelCase (e.g. PointData) here.

Reply:
`qpt_data_type`?